src/vdaf/prio3.rs

```diff
@@ -28,17 +28,22 @@ use std::io::Cursor;
 use std::iter::IntoIterator;
 use std::marker::PhantomData;
 
+// TODO Add domain separation tag to field element generation as required by spec.
+
+// TODO Add test vectors and make sure they pass.
+
 /// The count type. Each measurement is an integer in `[0,2)` and the aggregate is the sum.
-pub type Prio3Count64 = Prio3<Count<Field64>, Prio3Result<u64>>;
+// XXX Question for reviewer: What do you think about removing the "64" suffix from this type name?
+// (Here and below.)
+pub type Prio3Aes128Count64 = Prio3<Count<Field64>, Prio3Result<u64>, PrgAes128, 16>;
```

Comment:
Having `64` here is indeed potentially confusing since it's not clear if it's referring to the size of the underlying field or the size of the result. What's important is to make it clear to the caller what the range of values this metric can record is, so the naming convention depends on whether or not we ever expect to have `Prio3Aes128Count128` or `Prio3Aes128Count32`. The precedent I look to here is Prometheus. They have a single [Counter](https://prometheus.io/docs/concepts/metric_types/#counter) type and Prometheus' docs don't specify what type it is. If you look at [client libraries](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#Counter) it's clearly a 64 bit float. IMO if one-size-fits-all counters and gauges are good enough for Prometheus, they'll be good enough for us (though of course u64 has a much smaller range than float64).

Yeah, let's rename these to `Prio3Aes128Count`. TBH I think we could even do `Prio3Count`. Will we ever let the caller choose which PRG to use?

Reply:
Removed the "64" suffix from each of the names, but left the PRG name. I suspect users may one day have a preference for a different PRG.